docs/operator/installation/helm.md

```diff
@@ -68,8 +68,9 @@ You have to manually delete custom resource definitions created by the `helm ins
     kubectl delete crd vulnerabilityreports.aquasecurity.github.io
     kubectl delete crd configauditreports.aquasecurity.github.io
     kubectl delete crd ciskubebenchreports.aquasecurity.github.io
+    kubectl delete crd kubehunterreports.aquasecurity.github.io
```

Comment:
Could you also add `clusterconfigauditreports.aquasecurity.github.io` that will be automatically installed with Helm please? We've already added it on the main branch.

Reply:
Thank you for the convention heads-up. I've added the other missing crd to the list.